src/get-started/fwe/local-caching.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Local caching
+description: Learn how to persist data locally.
+prev:
+  title: Networking and data
+next:
+    title: JSON serialization
+---
+
+Now that you've learned about how to load data from servers
+over the network, your Flutter app should feel more alive.
+However, just because you *can* load data from remote servers
+doesn't mea you always *should*. Sometimes, it's better to
+re-render the data you received from the previous network
+request rather than repeat it and make your user wait until
+it completes again. This technique of retaining application
+data to show again at a future time is called *caching*, and
+this page covers how to approach this task in your Flutter app.
+
+## Introduction to caching
+
+At its most basic, all caching strategies amount to the same
+three-step operation, represented with the following pseudocode:
+
+```dart
+Data? _cachedData;
+
+Future<Data> getData() async {
```

Comment:
Something I missed, this should have been `Future<Data> get data async {` ;-)

Reply:
Updated. I debated making this a getter myself :)

Reply:
Looks like this final pass happened after the PR was merged. I'll open another one.